exporters/prometheus/exporter.go

```diff
@@ -0,0 +1,182 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheus
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata"
+)
+
+// Exporter is a Prometheus exporter.
+type Exporter struct {
+	reader   metric.Reader
+	registry *prometheus.Registry
+}
+
+// NewPrometheusExporter returns a Prometheus exporter.
+func NewPrometheusExporter(reader metric.Reader) (*Exporter, error) {
```

Comment:
nit: It would be nice to add options (even if we don't currently have any) to this function for future extensibility.